### DISCUSSION TITLE
Строгий фильтр в поле select при добавлении.

### DISCUSSION THREAD

**Discussion by lutikoff** [totumonline/totum-ru-issues-and-discussions#538] [Как это сделать?] [answered]
Добрый день.

Столкнулся с такой проблемой, поле select обращается к другой таблице там масса значений со схожими именами, типа 12 Name, 32 Name и так далее, поиск работает не по строгому соответствию и когда много значений искомое просто находится за пределами выбора.
Как сделать строгий фильтр ?
Спасибо.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Есть там такая штука как режимы поска: https://ru.docs.totum.online/user-guide#filter-search-types

Вам скорее всего нужен `~: 32 Name` или `^~: 32 Name`

Думали еще сделать преднастройку типа для поля и эта-же история должна была использоваться в Префильтрах и where параметрах выборки. Но это после платных лицензий.

**Скопирую сюда для информации:**

~~: — так обозначен режим по умолчанию и эту комбинацию символов можно не вводить перед строкой поиска:

Азот (20 счет) найдется если ввести Аз 20.

!~~: или !: — режим по умолчанию с исключением:

Если ввести !: Аз 20 то покажет все, кроме Азот (20 счет) и аналогичных.

~: — пробел как символ:

Азот (20 счет) найдется если ввести ~: от (20.

!~: — исключить + пробел как символ.

=: полное соответсвие всего значения:

Азот (20 счет) найдется если ввести =: Азот (20 счет).

!=: — инвертированное полное соответствие.

^: — значение начинается с значения поиска:

Азот (20 счет) найдется если ввести ^: Азот или ^: Азот 20.

!^: — инвертированное.

^~: — значение начинается + пробел как символ:

Азот (20 счет) найдется если ввести ^: Азот (20.

!^~: — инвертированное.

**Reply by lutikoff** [reply]
Спасибо большое!
